and-bible/app/src/main/java/net/bible/android/view/util/widget/TwoLine2TitleListItem.java

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
To me it looks like this class could be done more simply by inheriting from TwoLineListItem.  Or what do you think?